packages/ga4/src/index.ts

```diff
@@ -96,8 +114,8 @@ function getArguments(args: any[]): [string | undefined, IProps] {
  *
  * -------------------------------- */
 
-function getEventMeta({ type = '', event }: Pick<IProps, 'type' | 'event'>) {
-  const searchString = document.location.search;
+function getEventMeta({ type = '', event }: Pick<IProps, 'type' | 'event'>): [string, string][] {
```

Comment:
I can't seem to get these configs loaded into VSCode with the Prettier extension. I'll look for a CLI command soon. Might be worth adding as a pre-commit hook if it can be automated.